README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
The choice of license is tangential to this. The license states what rules you must comply with. The DCO is an explicit assertion by the user that their contribution complies with the license. IOW regardless of whether using GPL or MIT or Apache, the use of DCO signoff is still appropriate. As a point of reference OpenStack is under the Apache license and also uses the DCO.